ursapictura/ThriveWell-Server
# Setup

## Detailed Description
Setup file structure for Repository Pattern and Unit Testing. Created data Models

## How to Test
1.  pull down repo to local machine
2. verify file structure and Model key/value set-up.

## Motivation and Context
This will define the basic organizational structure for the project.

## Types of Changes
